frontend/src/fixtures/personalSchedulesFixtures.js

```diff
@@ -5,6 +5,12 @@ const personalSchedulesFixtures = {
         "description": "TestDescription",
         "quarter": "W08"
     },
+    onePersonalScheduleArr: [{
+        "id": 1,
+        "name": "TestName",
+        "description": "TestDescription",
+        "quarter": "W08"
```

Comment:
This is an example of what the JSON looks like that comes from the `/api/personalschedules/all` endpoint.   Your fixtures need to match this, or the output of some other actual endpoint on Swagger; otherwise the component won't be usable when you try to integrate it into a real page later on:

```
[
  {
    "id": 15,
    "user": {
      "id": 5,
      "email": "phtcon@ucsb.edu",
      "googleSub": "115856948234298493496",
      "pictureUrl": "https://lh3.googleusercontent.com/a/ALm5wu39rds75RezWgqD6WV4n38l2n0XL-1X6wETUeQU=s96-c",
      "fullName": "Phill Conrad",
      "givenName": "Phill",
      "familyName": "Conrad",
      "emailVerified": true,
      "locale": "en",
      "hostedDomain": "ucsb.edu",
      "admin": true
    },
    "name": "Conrad",
    "description": "Conrad's Courses",
    "quarter": "20222"
  },
  {
    "id": 16,
    "user": {
      "id": 5,
      "email": "phtcon@ucsb.edu",
      "googleSub": "115856948234298493496",
      "pictureUrl": "https://lh3.googleusercontent.com/a/ALm5wu39rds75RezWgqD6WV4n38l2n0XL-1X6wETUeQU=s96-c",
      "fullName": "Phill Conrad",
      "givenName": "Phill",
      "familyName": "Conrad",
      "emailVerified": true,
      "locale": "en",
      "hostedDomain": "ucsb.edu",
      "admin": true
    },
    "name": "Conrad",
    "description": "Conrad's W22 Courses",
    "quarter": "20221"
  }
]
```

If you are using a different endpoint that produces the JSON you have here, then I stand corrected... please let me know if that's the case.

This is the one I used:

<img width="1179" alt="image" src="https://user-images.githubusercontent.com/1119017/204154584-c62cfa05-a094-483d-8859-772b796babf5.png">